docs: establish Artifacts WG (#409)
cncf/tag-app-delivery: artifacts-wg/charter/appendix.md

# Artifacts WG Charter Appendix 

## Problem Statements
* As an artifact consumer I need to discover attributes about every component used or potentially used in my system. I need consistent practices and patterns for querying for these attributes across many artifact types.

> * As an artifact provider I need to deliver complete components to a runtime platform which may be completely disconnected from the environment where those components are developed. I want consistent practices for bundling and deploying different component types.

Reviewer: ```suggestion
* As an artifact provider, I need to deliver a complete set of components to a runtime platform which may be fully disconnected from the environment from where those components are developed or assembled. I want consistent practices for bundling and deploying different component types.
```

> * As a security analyst I need to understand and verify the content of bundles used in software and deployed on my platforms.

Reviewer: ```suggestion
* As a security analyst, I need to understand and verify the content of bundles used in software and deployed on my platforms.
```

> * [Proposed] As a platform operator I need a consistent pattern and implementation for deploying bundles to provide additional platform capabilities. For example, I need a consistent pattern for adding extension controller-managers (operators) to Kubernetes.

Reviewer: ```suggestion
* [Proposed] As a platform operator, I need a consistent pattern and implementation approach for deploying bundles to provide additional platform capabilities. For example, I need a defined pattern for adding extension controller-managers (operators) to Kubernetes.
```

Reviewer 2: I'm fine with these changes or the original. @sabre1041 in meetings we agreed that we're going to iterate on these use cases as one of the first tasks of the established WG.
